dist/listbox-button/listbox-button.css

```diff
@@ -44,12 +44,12 @@ button.expand-btn[aria-expanded="true"] ~ div.listbox-button__listbox,
 .listbox-button button[aria-invalid="true"] {
   border-color: var(--listbox-button-invalid-border-color, var(--color-stroke-attention));
 }
-.listbox-button button.btn--form {
+.listbox-button:not(.listbox-button--error) button.btn--form {
   border-color: var(--listbox-button-border-color, var(--color-stroke-default));
 }
-.listbox-button button.btn--form:hover,
-.listbox-button button.btn--form:focus,
-.listbox-button button.btn--form:active {
+.listbox-button:not(.listbox-button--error) button.btn--form:hover,
+.listbox-button:not(.listbox-button--error) button.btn--form:focus,
+.listbox-button:not(.listbox-button--error) button.btn--form:active {
```

Comment:
Hmm, Im wondering. Do we need the `:not`? 
The way I see it, if we do not have the error class then we should show the default color. If the error class is present, that should be a stronger style match and it should take precedence, right? 
```
.listbox-button button.btn--form:hover { }
.listbox-button.listbox-button--error button.btn--form:hover { }
```
That should be stronger right?
Just wondering so we can simplify things.

Reply:
Okay, lets leave it. I see what the issue is, we don't have a hover state for error. If we did that means we dont need this. 
Lets add a comment to those lines to remove them when we get an error state. Otherwise LGTM

Reply:
I realized that having a focus state that didn't change color was likely an `a11y` issue so I just reverted this and made the styling match textbox. We can apply the additional colors if/when we get them.